scripts/UFS_helpers/compile_ufs.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# A compile script for the UFS weather model - for data atmosphere only.
+
+set -eux
+
+cwd=$(pwd)
+UFSsrc=$cwd/../../sorc/ufs_model.fd/
+
+if [[ ! -d "${UFSsrc}" ]]; then
+  echo "Error: Source code path: '${UFSsrc}' does not exist."
+  echo "Fix your clone and try again."
+  exit 2
+fi
+
+echo " "
+echo " Building the UFS Weather Model "
+echo " Path to the source code: " ${UFSsrc}
+echo " "
+
+APP="NG-GODAS"
+CCPP_SUITES=""
+PDLIB="OFF"
+#
+# Valid only for WCOSS2; enable parallel restart I/O
+# TODO: Remove following option when parallel restart option _works_.
+# ----
+PARALLEL_RESTART="NO"
+
+EXEC_NAME="ufs_model.x"
+#
+# D O  N O T  E D I T  B E L O W
+
+cd ${UFSsrc} || false
+
+source "./tests/detect_machine.sh"
+source "./tests/module-setup.sh"
+
+MAKE_OPT="-DAPP=${APP} -D32BIT=ON -DCCPP_SUITES=${CCPP_SUITES}"
```

Comment:
```suggestion
MAKE_OPT="-DAPP=${APP}"
```
32BIT is an option for the FV3 dycore.  Has no bearing on the NG-GODAS app.